healthtools/tests/test_scrapers.py

```diff
@@ -23,110 +23,110 @@ def setUp(self):
         self.nhif_outpatient_cs_scraper = NhifOutpatientCsScraper()
 
     def test_it_gets_the_total_number_of_pages(self):
-        self.doctors_scraper.get_total_number_of_pages()
-        self.assertIsNotNone(self.doctors_scraper.num_pages_to_scrape)
+        self.doctors_scraper.set_site_pages_no()
+        self.assertIsNotNone(self.doctors_scraper.site_pages_no)
 
     def test_it_scrapes_doctors_page(self):
         entries = self.doctors_scraper.scrape_page(
-            "http://medicalboard.co.ke/online-services/retention/?currpage=1")
-        self.assertTrue(len(entries) == 60)
+            "http://medicalboard.co.ke/online-services/retention/?currpage=1", 5)
+        self.assertTrue(len(entries[1]) == 60)
 
     def test_it_scrapes_foreign_doctors_page(self):
         entries = self.foreign_doctors_scraper.scrape_page(
-            "http://medicalboard.co.ke/online-services/foreign-doctors-license-register/?currpage=1")
-        self.assertTrue(len(entries) == 60)
+            "http://medicalboard.co.ke/online-services/foreign-doctors-license-register/?currpage=1", 5)
+        self.assertTrue(len(entries[1]) == 60)
 
     def test_it_scrapes_clinical_officers_page(self):
         entries = self.clinical_officers_scraper.scrape_page(
-            "http://clinicalofficerscouncil.org/online-services/retention/?currpage=1")
-        self.assertTrue(len(entries) == 60)
+            "http://clinicalofficerscouncil.org/online-services/retention/?currpage=1", 5)
+        self.assertTrue(len(entries[1]) == 60)
 
     def test_it_scrapes_nhif_inpatient_page(self):
-        entries = self.nhif_inpatient_scraper.scrape_page(1)
+        entries = self.nhif_inpatient_scraper.scrape_page(1, 5)
         self.assertTrue(len(entries) > 1)
 
     def test_it_scrapes_nhif_outpatient_page(self):
-        entries = self.nhif_outpatient_scraper.scrape_page(1)
+        entries = self.nhif_outpatient_scraper.scrape_page(1, 5)
         self.assertTrue(len(entries) > 1)
 
     def test_it_scrapes_nhif_outpatient_cs_page(self):
-        entries = self.nhif_outpatient_cs_scraper.scrape_page(1)
+        entries = self.nhif_outpatient_cs_scraper.scrape_page(1, 5)
         self.assertTrue(len(entries) > 1)
 
     def test_doctors_scraper_uploads_to_elasticsearch(self):
-        with open(TEST_DIR + "/dummy_files/doctors.json", "r") as my_file:
+        with open(TEST_DIR + "dummy_files/doctors.json", "r") as my_file:
```

Comment:
@DavidLemayian back to this conversation, you see for the dummy data, it's included in the repo where as the data directory which is a substitute for the aws bucket, is ignored. That's why I strongly suggest having the dummy data not included in the data dir. Another work around would be to have the tests read from the archived data but as we know those are a lot of records that will delay the testing.